collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
I may have misread what you said. Is this what you are suggesting?

Move teams out of collector_item and into a new repository (handled similar to the scope repo). Continue to put all features in the feature repo but only load them on-demand based on UI selections as opposed to gathering everything at once. Have the UI create a collector item representing the config in a similar manor to how the git collector is handled.